worker/uniter/resolver/opfactory.go

```diff
@@ -112,8 +112,10 @@ func (s *resolverOpFactory) wrapHookOp(op operation.Operation, info hook.Info) o
 	switch info.Kind {
 	case hooks.ConfigChanged:
 		v := s.RemoteState.ConfigVersion
+		us := s.RemoteState.Series
```

Comment:
"series" instead of "us".... "us" made me stop and think what it was

Reply:
done